### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by translate-h5p**
Translations update from [Weblate](https://translate-h5p.tk/weblate/projects/h5p/h5p-agamotto/) for H5P/h5p-agamotto.

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by otacke — CHANGES_REQUESTED** [review]
@translate-h5p The af.json file contains many empty labels. Those need to contain the English original if a translation has not yet been added (or people submitted empty values).

File: language/af.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            "label": "Klank",
+            "description": "Opsionele klankbaan wat speel wanneer 'n prent vertoon word."
+          }
+        ]
+      }
+    },
+    {
+      "label": "Gedragsinstellings",
+      "description": "Hierdie opsies laat jou beheer uitoefen oor die uitvoering van die taak.",
+      "fields": [
```
Comment (otacke) [inline language/af.json:41]: @weblate Those missing labels would probably leave the users a little lost.

**Review by otacke — CHANGES_REQUESTED** [review]
@weblate Still some changes needed.

File: language/af.json (hunk 1 above)
Comment (sr258) [reply language/af.json:41]: Yes, there was a bug when updating Weblate that set some string to read only. I've fixed it and I'm giving Victor some time to finish the translations in Weblate.

Comment (otacke) [reply language/af.json:41]: Perfect! Then I'll just wait for the next push to this repository.

**Review by otacke — APPROVED** [review]
Thanks!

**Pull request merged**